README.md

```diff
@@ -1,14 +1,29 @@
+
 # ircb
 
 A versatile IRC bouncer.
 
 
 ## Setup
-- Install Python3 and Python3 development packages for your distro, if not already there.
-  In Fedora, it'd be: ``sudo dnf install -y python3 python3-devel``
-- Install dependencies: ``pip3 install -r requirements.txt``
-- Copy and extend ``ircb/config/default_settings.py``, as needed, to a custom
-  location. say, ``/etc/ircb/settings.py``.
+
+If you want to set up `ircb` for development purpose. Follow the given steps
+
+- Install Python3 and Python3 development packages for your distro, if not already there. In Fedora, it'd be:  
+``sudo dnf install -y python3 python3-devel python3-pip``
+
+- Install virtualenvwrapper:  
+``sudo pip3 install virtualenvwrapper``
```

Comment:
I reverted this step to the one we had earlier `sudo pip install virtualenvwrapper`. Yes we can install it via dnf
@rtnpro any reason on why you used pip instead of dnf earlier?